Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -9,6 +9,12 @@ final class DemoMerchantAPI {
 
     static let sharedService = DemoMerchantAPI()
     var accessToken: String?
+    
+    enum InjectedValues {
```

Comment:
Same comment as I left on Android [here](https://github.com/paypal/Android-SDK/pull/142#discussion_r1198030030) regarding adding some comments around making this more clear for less experienced devs to use.

Reply:
Added comment here adee603805692b65d9586ddc143a8b5e41b2a69b